### ISSUE TITLE
The performance of the model on the entire dataset

### ISSUE THREAD

**Issue by Liang-bk** [Vincent-ZHQ/MRDF#5]
Hello, I notice that only 10% of the whole dataset is used in the code, does it mean that acc is also based on this part of the data?
And I want to test the model on the whole dataset, but there are some problems happened while loading the ckpt model: Missing key(s) in state_dict: "video classifier.0.weight", "audio classifier.0.weight". Was that model structure conflicts in the code and ckpt model?

**Comment by Vincent-ZHQ** [comment]
Yes. We only use part of the original dataset. FakeAVCelebl is very imbalanced. So we randomly sample some samples and propose a five-fold cross-validation strategy for evaluation. For inference, you should ensure the checkpoint you use is consistent with the model you test.

**Comment by Liang-bk** [comment]
Thanks for answer, and the point I had missed in last question is that I download the ckpt model from the google drive in the readme.md, but I can't adapt it on the network in this code :(

**Comment by Vincent-ZHQ** [comment]
The provided chpts correspond to CE-based and Margin-based methods, respectively. The margin-based method has no audio and video classifier. I don't know about the details of your reported errors. I remember I made sure that the code could run before I uploaded.

**Comment by Liang-bk** [comment]
Thank you very much! I will try again.

**Comment by Amit507017** [comment]
Facing the same issue with the downloaded checkpoint. 

Missing key(s) in state_dict: "video classifier.0.weight", "audio classifier.0.weight". Was that model structure conflicts in the code and ckpt model?
